docker/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM node:12.18.3-buster
+
+#Set production mode deployment
+ENV NODE_ENV "Production"
+
+#copy the content of current directory to /app inside container
+COPY . /app
+
+WORKDIR /app 
+
+RUN npm install && npm run postinstall
```

Comment:
Not sure whether you really need `&& npm run postinstall`

Reply:
Because we are serving the production build of the FE from the docker. Since we are running `npm run build` in the next step, it needs all the dependencies in place. So `npm run postinstall` is required.

Reply:
As far as I know `npm postinstall` automatically runs after `npm install`. Are you sure it's not working?

Reply:
Yes, but when I tried without `postinstall`, before adding it, the build was breaking. So I added it, it worked fine.

Reply:
Figured out the issue, since docker build process runs as a super-user inside the container, it was causing permission issues with postinstall. Fixed by adding `--unsafe-perm` flag to `npm install`

Reply:
Fixed the issue in latest commit. Please check out.